> Solving the next problem at: 2024-09-27 08:05:16 UTC
> Search algorithm: bfs
> Instance ID=grid/pprob01-err-rate-0-5
> Plan length=14
> Domain class: grid
> Instance name: pprob01-err-rate-0-5
> Ground plan:
['(move node2-4 node1-4)', '(move node1-4 node1-3)', '(move node1-3 node1-2)', '(move node1-2 node0-2)', '(pickup node0-2 key3)', '(move node0-2 node1-2)', '(move node1-2 node1-3)', '(unlock node1-3 node2-3 key3 square)', '(move node1-3 node2-3)', '(pickup-and-loose node2-3 key0 key3)', '(move node2-3 node1-3)', '(move node1-3 node1-2)', '(move node1-2 node1-1)', '(putdown node1-1 key0)']
> Lifted plan:
['move', 'move', 'move', 'move', 'pickup', 'move', 'move', 'unlock', 'move', 'pickup-and-loose', 'move', 'move', 'move', 'putdown']
>>>  Vanilla ground repair length:
1

>>>  Vanilla ground repair:
pickup,(key ?key),effPos,1

An error occurred during A* search: name 'logger' is not defined
Traceback (most recent call last):
  File "/home/nader/projects/lifted-white-plan-domain-repair/instance_solver.py", line 70, in solve_instance
    path, goal_node = searcher.find_path(log_interval=log_interval)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nader/projects/lifted-white-plan-domain-repair/search_partial_grounding/search_algorithms.py", line 49, in find_path
    for neighbor in current_node.get_neighbors():
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nader/projects/lifted-white-plan-domain-repair/search_partial_grounding/node.py", line 167, in get_neighbors
    logger.debug(f'> Current state:\n{self.current_state}')
    ^^^^^^
NameError: name 'logger' is not defined. Did you mean: 'self.logger'?

> Time spent on this problem: 0.0 hours = 0.0 minutes = 0.49 seconds
> Subprocess finished.
